Library/Homebrew/cmd/test-bot.rb

```diff
@@ -91,6 +92,13 @@ def failed?
 
     def puts_command
       cmd = @command.join(" ")
+      if ENV["TRAVIS"]
+        puts cmd
+        @travis_timer_id = rand(2**32).to_s(16)
+        puts "travis_time:start:$travis_timer_id:#{@travis_timer_id}"
+        puts "travis_fold:start:#{@command.join(".")}"
+        return
+      end
       print "#{Tty.blue}==>#{Tty.white} #{cmd}#{Tty.reset}"
```

Comment:
This is only used for the travis inline command so I think it's more consistent as-is. At least the `==>` looked pretty weird.